app/apphandlers/embeddedAppHandler.go

```diff
@@ -139,13 +139,21 @@ func (a *embeddedApp) StartRudderCore(ctx context.Context, options *app.Options)
 		FeaturesRetryMaxAttempts: 10,
 	})
 
+	dbPool, err := misc.GetDatabaseConnectionPool(ctx, config, statsFactory, "jobsdb")
```

Comment:
```suggestion
	dbPool, err := misc.GetDatabaseConnectionPool(ctx, config, statsFactory, "embedded-app")
```

I would avoid using `jobsdb` for anything else other than jobsdb component

Reply:
makes sense.
Should we suffix with `app`?
`embedded`, `processor`, `gateway` look good enough no?

Reply:
processor or gateway can be a bit ambiguous by themselves.